src/main/kotlin/no/nav/syfo/application/IVarselRepository.kt

```diff
@@ -2,11 +2,9 @@ package no.nav.syfo.application
 
 import no.nav.syfo.domain.PersonIdent
 import no.nav.syfo.domain.Varsel
-import no.nav.syfo.domain.Vurdering
 
 interface IVarselRepository {
-    fun getUnpublishedVarsler(): List<Pair<PersonIdent, Varsel>>
+    fun getUnpublishedVarsler(): List<Triple<PersonIdent, String, Varsel>>
```

Comment:
Skulle vi hatt en [inline value class](https://kotlinlang.org/docs/inline-classes.html) for journal post id'en her? 😊

Reply:
Ja, det kunne vi kanskje prøvd, sjekker det ut :+1:

Reply:
Endrer til å sjekke at alle tegn er tall i stedet  - `value.all { it.isDigit() }` - blir litt tydeligere.